source/development/packaging/generate-dashboard-package.rst

```diff
@@ -144,119 +148,120 @@ Using the script
 
 Run the ``build-packages.sh`` script in the ``dev-tools/build-packages/`` folder of the repository. The script requires the following parameters:
 
--  ``-v``: Version of the package.
--  ``-r``: Revision of the package.
--  ``--deb`` or ``--rpm``: Distribution of the package.
--  ``-a``: Path to the ``wazuh-package.zip``.
--  ``-s``: Path to the ``security-package.zip``.
--  ``-b``: Path to the ``dashboard-package.zip``.
+- ``-a``, ``--app``: Path to the ``wazuh-package.zip``.
+- ``-b``, ``--base``: Path to the ``dashboard-package.zip``.
+- ``-s``, ``--security``: Path to the ``security-package.zip``.
+- ``-v``, ``--version``: Set the version of this build.
+- ``--all-platforms``: Build all platforms.
+- ``--deb``: Build deb.
+- ``--rpm``: Build rpm.
+- ``--tar``: Build tar.gz.
+- ``-r``, ``--revision``: [Optional] Set the revision of this build. By default, it is set to 1.
+- ``--production``:[Optional] The naming of the package will be ready for production. Otherwise, it will include the hash of the current commit.
+- ``--arm``: [Optional] Build for arm64 instead of x64.
+- ``--debug``: [Optional] Enables debug mode, which will show detailed information during the script run.
+- ``--silent``: [Optional] Enables silent mode, which will show the minimum possible information during the script run. ``--debug`` has priority over this.
```

Comment:
```suggestion
- ``-a``, ``--app``: Indicate the path to the ``wazuh-package.zip``.
- ``-b``, ``--base``: Indicate the path to the ``dashboard-package.zip``.
- ``-s``, ``--security``: Indicate the path to the ``security-package.zip``.
- ``-v``, ``--version``: Set the version for this build.
- ``--all-platforms``: Build the package for all platforms.
- ``--deb``: Build a deb package.
- ``--rpm``: Build an rpm package.
- ``--tar``: Build a tar.gz archive.
- ``-r``, ``--revision``: [Optional] Indicate the revision for this build; defaults to 1.
- ``--production``:[Optional] Prepare the package naming for production; otherwise, it will include the hash of the current commit.
- ``--arm``: [Optional] Build for arm64 architecture instead of x64.
- ``--debug``: [Optional] Enables debug mode to display detailed information during script execution.
- ``--silent``: [Optional] Enables silent mode to show minimal information during the script run; note that --debug takes precedence over this option.
```